index.js

```diff
@@ -137,21 +150,68 @@ function useCard()
     //Play card if valid move, otherwise ignore
     if (isValidCard == true)
     {
+        let cardBeingPlayed = players[gameTurn].playerDeck.getCard(cardIndex);
         alert("Debug: Valid move.");
+        
+        
+        //Will run if there is a stackable card played, +2 or +4
+        if(drawStack.stackAmt != 0){
+                if(cardBeingPlayed.value != drawStack.cardValue){
+                    alert("Card chosen Doesn't stack");
+                    return;
+                }else if(cardBeingPlayed.value == 1 && cardBeingPlayed.color != 'Special'){
+                    alert("Card chosen Doesn't stack");
+                    return;
+                }
+                
+        }
+        
         players[gameTurn].playerDeck.playCard(cardIndex);
-        gameTurn++;
+        
+        if(cardBeingPlayed.color == 'Special'){
+            if(cardBeingPlayed.value == 0){
+                cardWild();
+            }else if(cardBeingPlayed.value == 1){
+                cardDraw4();
```

Comment:
This card has a bug. When used, the number on the screen is -1 and no card is valid to play except another one of these cards. Image: https://pasteboard.co/I8TAXG4.png